src/views/Opportunities/OpportunityComponents/OpportunityForm.jsx

```diff
@@ -223,6 +224,19 @@ const OpportunityForm = (props) => {
     });
   };
 
+    if (props.id) {
+      const searchedemployed = employerOptions.find((option) => option.label === props.employer.name);
+      if (!searchedemployed) {
+        const newvalue = {
```

Comment:
this is not a proper variable name. Please rename to a more meaningful one.

Reply:
I rename this searchedemployed to    searchEmployer